pkgs/tools/typesetting/tex/pplatex/default.nix

```
@@ -0,0 +1,47 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkgconfig,
+  pcre,
+} :
+  stdenv.mkDerivation rec {
+    pname = "pplatex";
+    rev = "5cec891ad6aec0115081cdd114ae1cc4f1ed7c06";
+
+    version = "1.0-pre-${builtins.substring 0 6 rev}";
```

If you decided to switch to the latest commit instead of the tag, then the convention for the version is described here https://nixos.org/nixpkgs/manual/#sec-package-naming
```suggestion
    version = "unstable-2015-09-14";
```

---

I decided because 23 commits  (some windows specific, but still) on HEAD vs 13 commits in rc2. Fixed